.editorconfig

```diff
@@ -20,3 +22,9 @@ insert_final_newline = false
 
 [*.md]
 trim_trailing_whitespace = false
+
+[*.go]
+indent_size = 8
+
+[*.{tmpl, html, less, js}]
+indent_size = 4
```

Comment:
For javascript most defaults to 2 spaces identing

Reply:
I'd remove this as well and reindent matching files in `docs`. The other formats should already be at 2-space outside that folder.